src/phoenix/server/api/dataloaders.py

```diff
@@ -0,0 +1,54 @@
+from collections import defaultdict
+from typing import AsyncContextManager, Callable, DefaultDict, List, Optional, Tuple
+
+from ddsketch import ddsketch
+from sqlalchemy import and_, select
+from sqlalchemy.ext.asyncio import AsyncSession
+from strawberry.dataloader import DataLoader
+
+from phoenix.db import models
+from phoenix.server.api.input_types.TimeRange import TimeRange
+
+
+class LatencyMsQuantile(DataLoader[Tuple[str, Optional[TimeRange], float], Optional[float]]):
```

Comment:
```suggestion
class LatencyMsQuantileDatLoader(DataLoader[Tuple[str, Optional[TimeRange], float], Optional[float]]):
```

Reply:
might be good to have a bit of a naming convention.

Reply:
ok